recipes/fmt/all/conanfile.py

```diff
@@ -71,44 +63,48 @@ def package_id(self):
             del self.info.options.with_fmt_alias
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version],
-                  destination=self._source_subfolder, strip_root=True)
-
-    def _configure_cmake(self):
-        if self._cmake:
-            return self._cmake
-        self._cmake = CMake(self)
-        self._cmake.definitions["FMT_DOC"] = False
-        self._cmake.definitions["FMT_TEST"] = False
-        self._cmake.definitions["FMT_INSTALL"] = True
-        self._cmake.definitions["FMT_LIB_DIR"] = "lib"
+        tools.get(**self.conan_data["sources"][self.version], destination=self.source_folder, strip_root=True)
+
+    def layout(self):
+        cmake_layout(self)
+
+    def generate(self):
+        cmake_toolchain = CMakeToolchain(self)
```

Comment:
🟨 https://github.com/conan-io/conan-center-index/pull/4838#discussion_r625589456

Last time the team's answer was too experimental.
@danimtb it's been almost four months... has the answer changed?

Reply:
Hi, @prince-chrismc! Yes, I'm a new Conan team member, nice to meet you.

The new CMake toolchain is now much more mature than it was on March, and, although it's still experimental, we want to start allowing it in Conan Center Index soon. That's why we created this `new-cmake-toolchain` branch, to migrate a bunch of recipes and do some bug-hunting before removing the experimental tag.

This is the first PR of many to come to this branch, but it will probably be a while before we merge it to master.

Reply:
Welcome! Thanks for the small explanation, that's makes a lot of sense.